build/Program.cs

```diff
@@ -50,8 +50,10 @@ public enum ProjectBuildType
         new("Unity.IL2CPP", "win-x64"),
         new("Unity.IL2CPP", "linux-x64"),
         new("Unity.IL2CPP", "macos-x64"),
-        new("NET.Framework", "win-x64"),
-        new("NET.CoreCLR", "win-x64")
+        new("NET.Framework", "win-x86", "net40"),
+        new("NET.Framework", "win-x86", "net452"),
+        new("NET.CoreCLR", "win-x64", "netcoreapp3.1"),
+        new("NET.CoreCLR", "win-x64", "net6")
```

Comment:
Does it even make sense for these to have an RuntimeIndentifier?

Reply:
For NetFX it does because most games will refuse to run if the bitness is incorrect, as the launcher defines the bitness of the entire process. I'm only sticking to the most popular configurations I've seen/I know about.

For NET.CoreCLR it's a different story, but in my future branch I've also merged the two projects so it's also irrelevant